From 797fe9ae1c90ae9b727b9bbcb452176cb5c6afcf Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Thu, 14 Feb 2019 17:12:29 +0800
Subject: [PATCH] PCI/portdrv: drop unused PCIE_PORT_SERVICE_VC

Based on the previous commit 168f3ae595 (PCI/portdrv:
Remove unused PCIE_PORT_SERVICE_VC)

The PCIE_PORT_SERVICE_VC has been dropped, remove the vestigial code
to avoid building issue.

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/pci/pcie/portdrv_core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pci/pcie/portdrv_core.c b/drivers/pci/pcie/portdrv_core.c
index e02e639..4473e5c 100644
--- a/drivers/pci/pcie/portdrv_core.c
+++ b/drivers/pci/pcie/portdrv_core.c
@@ -189,7 +189,7 @@ static int pcie_init_service_irqs(struct pci_dev *dev, int *irqs, int mask)
 		if (dev->irq)
 			irq = dev->irq;
 		for (i = 0; i < PCIE_PORT_DEVICE_MAXSERVICES; i++)
-			if (irqs[i] == -1 && i != PCIE_PORT_SERVICE_VC_SHIFT)
+			if (irqs[i] == -1)
 				irqs[i] = irq;
 		return 0;
 	}
-- 
1.7.9.5

